base_link: 'world'
# =====================================================
gripper:
    end_effector_name: 'gripper'  #ee group name

    #actuated joints in end effector
    joints : ['jaco2_joint_finger_1','jaco2_joint_finger_2','jaco2_joint_finger_3']

    #open position
    pregrasp_posture : [0.4, 0.4, 0.4]
    #xopregrasp_posture : [0.0, 0.0, 0.0]
    pregrasp_time_from_start : 4.0

    #close position
    grasp_posture : [0.697, 0.697, 0.697]
    grasp_time_from_start : 4.0

    #distance from wrist joint to palm of end effector
    grasp_pose_to_eef_translation :  [0, 0, 0.1]

    #Rotation from wrist joint to std end effector orientation
    #z-axis pointing toward object to grasp
    #x-axis perp. to movement of grippers
    #y-axis parallel to movement of grippers
    grasp_pose_to_eef_rotation : [0, 3.14159, 0]


    # length of grippers (distance from finger tip to inner palm)
    finger_to_palm_depth : 0.11